### PR TITLE
Implementing Travis tests

### PR DESCRIPTION
**Pull request by jimbo8098**
Adding basic travis tests to ensure the repo can build deps

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: ruby
+before_install:
+  - gem install bundler
```
Comment (gsdevme) [inline .travis.yml]: worth using the makefile given we have one, so `make bundle`

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+      solve (~> 4.0)
+      thor (>= 0.20)
+    builder (3.2.4)
+    chef (15.7.32)
+      addressable
+      bcrypt_pbkdf (~> 1.0)
+      bundler (>= 1.10)
+      chef-config (= 15.7.32)
+      chef-utils (= 15.7.32)
+      chef-zero (>= 14.0.11)
```
Comment (gsdevme) [inline Gemfile.lock]: We maybe want to peg this to chef 13 though (at least right now).

Comment (gsdevme) [reply Gemfile.lock]: I guess there isn't any chef tests in here so doesn't matter too much

**Comment by jimbo8098** [comment]
Both fixed in the last commit! Lemme know what you think.

**Pull request merged**